ci-operator/step-registry/cucushift/upgrade/rhel-worker/cucushift-upgrade-rhel-worker-commands.sh

```diff
@@ -242,6 +300,7 @@ echo -e "The source release version is gotten from clusterversion resource, that
 if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
     run_command "oc get node -owide"
     rhel_repo
+    rhel_pre_upgrade
```

Comment:
I guess we also can apply the step to `openshift-upgrade-qe-sanity-disconnected`, `openshift-upgrade-qe-sanity` and `openshift-upgrade-qe-test-disconnected`

Reply:
@jianlinliu I see the step `cucushift-upgrade-rhel-worker` is already included in the above three chains? 
>openshift-upgrade-qe-sanity-disconnected, openshift-upgrade-qe-sanity and openshift-upgrade-qe-test-disconnected

Reply:
Ah, my fault, I mis-read the chains.